charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -45,6 +45,13 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
     verbs: ["get", "watch", "list"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get", "watch", "list"]
+{{- else }}
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get"]
```

Comment:
Why do we need get if webhook is disabled? Just for the post install hook? Can you add a comment for this?

Reply:
Yeah, we need this get for the post install hook